apps/web/vibes/soul/sections/featured-blog-post-list/index.tsx

```diff
@@ -11,15 +12,10 @@ interface Props {
   title: string;
   description?: string;
   cta?: Link;
-  posts: BlogPost[];
+  posts: Streamable<BlogPost[]>;
 }
 
-export const FeaturedBlogPostList = function FeaturedBlogPostList({
-  title,
-  description,
-  cta,
-  posts,
-}: Props) {
+export function FeaturedBlogPostList({ title, description, cta, posts: streamablePosts }: Props) {
```

Comment:
🍹 no need to alias to `streamablePosts` since we pass it straight down to `BlogPostList`. The reason we usually alias it is to work around the `no-shadow` ESLint rule when it complains we name the streamed result the same as the prop (i.e., having a prop `posts`, but then also naming the streamed value in the `Streamable` callback `posts`)

Reply:
@migueloller Ahh, makes sense! I'll update this real quick